go.mod

```diff
@@ -17,19 +18,17 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.12
+	github.com/tendermint/tendermint v0.34.13
```

Comment:
Can we hold off on this? It was reported to be causing some issues when used on mainnet.

Reply:
I thought tendermint 0.34.12 has problems. `..13` is needed for IAVL update, which has good performance improvements.

Reply:
you are right, Bez confirmed.